.github/workflows/combine_nb_to_docs_testing.sh

```diff
@@ -1,22 +1,18 @@
-rm -rf all_tools.ipynb
 rm -rf break.md
 
-# Combined notebook flow - will be tested
-nbmerge quickstart.ipynb logging.ipynb feedback_functions.ipynb >> all_tools.ipynb
-
 # Colab quickstarts
 nbmerge colab_dependencies.ipynb langchain_quickstart.ipynb >> langchain_quickstart_colab.ipynb
 nbmerge colab_dependencies.ipynb llama_index_quickstart.ipynb >> llama_index_quickstart_colab.ipynb
 nbmerge colab_dependencies.ipynb no_framework_quickstart.ipynb >> no_framework_quickstart_colab.ipynb
 
 # Create pypi page documentation
-jupyter nbconvert --to markdown all_tools.ipynb
-printf  "\n\n" >> break.md
-cat intro.md break.md all_tools.md > README.md
```

Comment:
double checking again; now we are removing all code from the trulens_eval/README as well?

Reply:
correct, replacing with links to all quickstarts. Emphasizing that TruLens is cross-framework.